src/bors/handlers/workflow.rs

```diff
@@ -269,6 +269,8 @@ async fn maybe_complete_build(
     };
 
     db.update_build_status(&build, status).await?;
+    db.update_build_duration(&build, payload.running_time)
```

Comment:
Let's modify the duration in the `db.update_build_status` call, to avoid doing two back-to-back SQL queries unnecessarily. I plan to extend the `update_build_status` function in the near future anyway.

Reply:
That's fine, it can just be `Option<Duration>`.